modules/web-console/e2e/testcafe/fixtures/admin/assume-identity.js

```diff
@@ -61,3 +91,82 @@ test('Become user', async(t) => {
         // See https://ggsystems.atlassian.net/browse/GG-24068
         .expect(userMenu.menuItem.withText('Admin panel').count).eql(1);
 });
+
+test("Action availability", async(t) => {
+    const _changeUserSelection = async(userName) => {
+        await t.click(admin.userNameCell.withText(userName))
+            .hover(admin.usersTable.actionsButton);
+    };
+
+    await t.navigateTo(resolveUrl('/settings/admin'))
+        .hover(admin.usersTable.actionsButton);
+
+    await _checkActionSet(t, true, true);
+
+    await admin.usersTable.performAction('Add user');
+
+    const createUserDialog = Selector('.modal-dialog');
+
+    await t.expect(createUserDialog.find('.modal-header').withText('Create User').exists).ok()
+        .click(createUserDialog.find('button').withText('Cancel'))
+        .hover(admin.usersTable.actionsButton);
+
+    await _checkActionSet(t, true, true);
+
+    await _changeUserSelection('John Doe');
+
+    await _checkActionSet(t, true, false);
+
+    await _changeUserSelection('John Doe');
+
+    await _checkActionSet(t, true, true);
+
+    await _changeUserSelection('User Name');
+
+    await _checkActionSet(t, false, false);
+
+    await _changeUserSelection('User Name');
+
+    await _checkActionSet(t, true, true);
+
+    await _changeUserSelection('User Name');
+
+    await admin.usersTable.performAction('Revoke admin');
+
+    await t.expect(successNotification.withText('Admin rights was successfully revoked for user: "User Name"').exists).ok();
+
+    await admin.usersTable.performAction('Grant admin');
+
+    await t.expect(successNotification.withText('Admin rights was successfully granted for user: "User Name"').exists).ok();
```

Comment:
There's no validation that admin permissions were granted/revoked - alert does not guarantee that

Reply:
Added check of admin status change.